pkg/gui/custom_commands.go

```diff
@@ -118,16 +118,22 @@ func (gui *Gui) menuPrompt(prompt config.CustomCommandPrompt, promptResponses []
 	return gui.createMenu(title, menuItems, createMenuOptions{showCancel: true})
 }
 
-func (gui *Gui) GenerateMenuCandidates(commandOutput string, filter string, format string) ([]string, error) {
+func (gui *Gui) GenerateMenuCandidates(commandOutput string, filter string, tFormat string, dFormat string) ([]string, []string, error) {
```

Comment:
In go you don't have to define an argument type if the next argument has the same type.
In lazygit we don't widely use it as you can debate over if it makes the function arguments more readable but for functions with many arguments where the function is getting wide it's better to use. 

```suggestion
func (gui *Gui) GenerateMenuCandidates(commandOutput, filter, tFormat, dFormat string) ([]string, []string, error) {
```